src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/Common/InvokeRestMethodCommand.Common.cs

```diff
@@ -278,7 +280,7 @@ private static bool TryConvertToXml(string xml, out object doc, ref Exception ex
             return doc != null;
         }
 
-        private static bool TryConvertToJson(string json, out object obj, ref Exception exRef)
+        private static bool TryConvertToJson(string json, out object? obj, ref Exception? exRef)
```

Comment:
[NotNullIfNotNull](https://learn.microsoft.com/en-us/dotnet/api/system.diagnostics.codeanalysis.notnullifnotnullattribute)

Reply:
obj is not null if true.

Reply:
done